frontend/apps/reader/modules/readertypography.lua

```diff
@@ -12,10 +12,12 @@ local C_ = _.pgettext
 local T = require("ffi/util").template
 local Screen = Device.screen
 
+local ReaderTypography = InputContainer:new{}
+
 -- This is used to migrate old hyph settings, and to show the currently
 -- used hyph dict language in the hyphenation menu.
 -- It will be completed with info from the LANGUAGES table below.
-local HYPH_DICT_NAME_TO_LANG_NAME_TAG = {
+ReaderTypography.HYPH_DICT_NAME_TO_LANG_NAME_TAG = {
```

Comment:
Might need some comment that the migration is done in onetime_migration.lua.
If I want to refactor all this and rename variables, it's not obvious from here that these variables are used elsewhere remote.